docs/Downloaders/SABnzbd/Basic-Setup.md

```diff
@@ -80,17 +83,18 @@ Covered and fully explained in [SABnzbd - Paths and Categories](/Downloaders/SAB
 
 ## Switches
 
-[Sabnzbd Documentation](https://sabnzbd.org/wiki/configuration/4.0/switches){:target="_blank" rel="noopener noreferrer"}
+[Sabnzbd Documentation](https://sabnzbd.org/wiki/configuration/4.3/switches){:target="_blank" rel="noopener noreferrer"}
 
 ### Queue
 
 `Settings` => `Switches` => `Queue`
 ![!Switches: Queue](images/sabnzbd-switches-queue.png)
 
+1. If you have really good indexers that get nzb from post not generated then some people might want Propagation delay  set to 5 min. so your not trying to grab a nzb 0 mins from posting and if your not getting it from the same server the poster used you might wrongly have article missing since it hasn't propagated to your server.. or if you use a reseller it may take them longer to get it from their upstream.
 1. When during download it becomes clear that too much data is missing, abort the job. and makes sure Sonarr/Radarr will get a notification and can look for another release.
 1. Since we have the .nzb history folder, you can decide what you want to do here w/ duplicate downloads. Mine is set to Tag job, but Pause or Fail job may make sense too.
 1. In case of "Pause", you'll need to set a password and resume the job. or you set it to "Abort" and Sonarr/Radarr can look for another release.
-1. Suggest this sort so that the most likely to still be there stuff is downloaded first.
+1. This should be set to Default. unless you know what your doing, as if someone has a decent size queue you have sab every 30s sort which could cause spikes in cpu due to it, let alone shuffling jobs around that may be in middle of actions so moving something in front of it could cause that job to take even longer to extract/stall waiting for next update as with sab by default you can only have 3 unpackers going (configurable).
```

Comment:
```suggestion
1. This should be set to the default unless you know what you are doing. Suppose you have a decent size queue, for example, in that case, you have sab sort every 30s, which could cause spikes in CPU, let alone shuffling jobs around that may be in the middle of actions. If this results in the jobs' order moving to the front it could cause that job to take even longer to extract/stall while waiting for the next update; as with sab, by default, you only have 3 unpackers going simultaneously (configurable).